.github/workflows/pr-cleanup.yml

```diff
@@ -0,0 +1,24 @@
+name: Pull Request Cleanup
+on:
+  pull_request_target:
+    types: [closed]
+
+jobs:
+  cleanup:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          ref: gh-deploy
+
+      - name: Remove Deployment
```

Comment:
Make sure it must not run if the PR is closed / merged before it got apporved

Reply:
fixed